packages/typespec-test/test/openai_generic/generated/typespec-ts/src/rest/models.ts

```diff
@@ -5,8 +5,13 @@ export interface CreateTranscriptionRequest {
   /**
    * The audio file object (not file name) to transcribe, in one of these formats: flac, mp3, mp4,
    * mpeg, mpga, m4a, ogg, wav, or webm.
+   *
+   * NOTE: The following type 'File' is part of WebAPI and available since Node 20. If your Node version is lower than Node 20.
+   * You could leverage our helpers 'createFile' or 'createFileFromStream' to create a File object. They could help you specify filename, type, and others.
    */
   file: string;
+    | NodeJS.ReadableStream
+    | File;
```

Comment:
Is this a valid syntax ?

Reply:
also why the change is different than the openai_modular case

Reply:
it is the same and the code you saw is what i resolved conflict manually. after regeneration it is the same now.